WalletWasabi.Tests/UnitTests/WabiSabi/Backend/CoinJoinMempoolManagerTests.cs

```diff
@@ -0,0 +1,77 @@
+using Moq;
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.CompilerServices;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using WalletWasabi.BitcoinCore.Mempool;
+using WalletWasabi.Services;
+using WalletWasabi.Tests.Helpers;
+using WalletWasabi.WabiSabi.Backend;
+using WalletWasabi.WabiSabi.Backend.Rounds.CoinJoinStorage;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.WabiSabi.Backend;
+
+public class CoinJoinMempoolManagerTests
+{
+	[Fact]
+	public async Task CoinJoinVisibleTestAsync()
+	{
+		var coreNode = await TestNodeBuilder.CreateAsync(nameof(CoinJoinMempoolManagerTests), nameof(CoinJoinVisibleTestAsync));
```

Comment:
It would be so so good for Wasabi development if we could remove the `TestNodeBuilder` and create UTs instead of setting up a whole infrastructure for testing something like this (and the other things). 

I know it is OOS for this PR but it's just like a mental note for myself: remove P2pNode dependency!

Reply:
Yes.